detection_status.py

```diff
@@ -0,0 +1,151 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+[
+    ReportData(
+        detectionStatus=1,
+        checkerMsg='Division by zero',
+        severity=40,
+        bugPathLength=1,
+        reviewData=ReviewData(status=0, comment=None, date=None,
+                              author=None),
+        column=13,
+        bugHash='e8f47588c8095f02a53e338984ce52ba',
+        checkedFile='/home/gamesh411/projects/codechecker/build/workspace/detection_status-9OoyLg/test_files/main.cpp'
+            ,
+        checkerId='core.DivideZero',
+        detectedAt='2019-12-06 15:28:25.964000',
+        reportId=10,
+        fixedAt='2019-12-06 15:28:28.210668',
+        details=None,
+        line=4,
+        fileId=2,
+        runId=1,
+        ),
+    ReportData(
+        detectionStatus=2,
+        checkerMsg="suspicious usage of 'sizeof(K)'; did you mean 'K'?"
+            ,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [continuation line unaligned for hanging indent](https://app.codacy.com/app/CodeChecker/codechecker/pullRequest?prid=4565982)